File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/BaseResponseContent.java
package de.dimedis.mobileentry.backend.response;

import android.text.TextUtils;

import com.google.gson.annotations.SerializedName;

import de.dimedis.mobileentry.backend.BackendService;

public class BaseResponseContent {
    @SerializedName("status")
    public String status;
    @SerializedName("status_descr")
    public String statusDescription;
    @SerializedName("user_message")
    public String userMessage;

    public boolean isStatusSuccess() {
        return TextUtils.equals(status, BackendService.STATUS_SUCCESS);
    }

    @Override
    public String toString() {
        return "BaseResponseContent{" +
                "status='" + status + '\'' +
                ", statusDescription='" + statusDescription + '\'' +
                ", userMessage='" + userMessage + '\'' +
                '}';
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/BaseResponseContent.kt
package de.dimedis.mobileentry.backend.response

import android.text.TextUtils
import com.google.gson.annotations.SerializedName
import de.dimedis.mobileentry.backend.BackendServiceConst.STATUS_SUCCESS

open class BaseResponseContent {
    @SerializedName("status")
    var status: String? = null

    @SerializedName("status_descr")
    var statusDescription: String? = null

    @SerializedName("user_message")
    var userMessage: String? = null

    fun isStatusSuccess(): Boolean {
        return TextUtils.equals(status, STATUS_SUCCESS)
    }

    override fun toString(): String {
        return "BaseResponseContent{" +
                "status='" + status + '\'' +
                ", statusDescription='" + statusDescription + '\'' +
                ", userMessage='" + userMessage + '\'' +
                '}'
    }
}